src/tests/unit/test_bitset.cpp

```diff
@@ -426,7 +426,7 @@ TYPED_TEST_P(BitsetTest, FromStream) {
 // SGI Extensions:
 // We don't test _Unchecked_XXX methods, as they're used inside of standard ones
 TYPED_TEST_P(BitsetTest, FindFirst) {
-#if defined(__GNUG__) && !defined(__clang__)
```

Comment:
Maybe it's better to change it's name from `FindFirst` to `FindFirst_libstdcxxExtensions`
I think test reports will be cleaner this way